packages/plugins/plugin-mixpanel/src/MixpanelPlugin.ts

```diff
@@ -91,4 +91,10 @@ export class MixpanelPlugin extends DestinationPlugin {
       this.mixpanel?.flush();
     }
   }
+
+  reset(): void {
+    if (this.isInitialized()) {
+      this.mixpanel?.reset();
```

Comment:
since we already to the optional chaining do we need to check for `this.isInitialized()` ?

Reply:
good point! I guess we also don't need it for `flush()`